src/utils/Fetch.ts

```diff
@@ -14,23 +14,21 @@
  */
 
 import * as http from 'http';
+import { RequestOptions } from 'http';
```

Comment:
Can we use `http.RequestOptions` below and remove this import?

Reply:
Or we could specify all the subcomponent we need from 'http'. Either one would be good IMO.